examples/gallery/gallery.slint

```diff
@@ -6,6 +6,9 @@ import { AboutPage, ControlsPage, ListViewPage, TextEditPage } from "ui/pages/pa
 import { GallerySettings } from "ui/gallery_settings.slint";
 import { SideBar } from "ui/side_bar.slint";
 
+import "../carousel/ui/fonts/Roboto-Regular.ttf";
+import "../carousel/ui/fonts/Roboto-Bold.ttf";
```

Comment:
This part is surely controversial - I'm aware of that. So happy to discuss options.

An alternative would be to move the font out of carousel and into some top-level directory and reference it from there.

Reply:
yeah, this doesn't really scale if every user needs to be doing that. 

The gallery is supposed to show the default widgets we have and their appearence by default.
So doing hack like that in the gallery is not great.